spec/views/curation_concerns/base/_form_progress.html.erb_spec.rb

```diff
@@ -64,4 +66,17 @@
       end
     end
   end
+
+  context "when the work has been saved before" do
+    before do
+      allow(work).to receive(:new_record?).and_return(false)
+      assign(:form, form)
+    end
+
+    let(:work) { stub_model(GenericWork, id: '456') }
```

Comment:
@jcoyne Updated to use `stub_model(GenericWork)`. This was a bit tricky since I had to also move the `assign(:form, form)` inside each different `context` for it to use the different works.